Feature/AB#15346 Org Info & AB#23421 Signing Authority

don-aot opened the pull request:
[AB#15346](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/15346):Organization information accessible in the same place and editable,
[AB#23421](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/23421):Signing Authority Contact Info

AndreGAot commented:
@don-aot  - Hi Don, have confirmed that the signing authority belongs to the application

don-aot commented:
> @don-aot - Hi Don, have confirmed that the signing authority belongs to the application

Updated the code and moved Signing Authority to the Application Table.

AndreGAot commented on applications/Unity.GrantManager/src/Unity.GrantManager.Domain.Shared/GrantApplications/ProjectInfoOptionsList.cs:
```diff
@@ -27,4 +27,21 @@ public static class ProjectInfoOptionsList
+              new KeyValuePair<string, string>("CORPORATION", "Corporation"),
+              new KeyValuePair<string, string>("PARTNERSHIP", "Partnership"),
+              new KeyValuePair<string, string>("INCORPORATED_COOPERATIVE", "Incorporated Cooperative"),
+              new KeyValuePair<string, string>("FIRSR_NATION", "First Nation or Indigenous-Owned Enterprises"),
```
spelling FIRST_NATION ?

don-aot replied:
> spelling FIRST_NATION ?

@AndreGAot  Corrected the spelling mistake.

AndreGAot commented:
@don-aot - please can we add some unit tests, have left 1 other comment, other testing good so far

don-aot commented:
> @don-aot - please can we add some unit tests, have left 1 other comment, other testing good so far

Pushed the unit testing for the new component

AndreGAot commented:
@don-aot - please can you check the Sector / Subsector fields, they are not behaving correctly, they look correct at the API level so think it is in the page / viewmodel and JS

don-aot commented:
> @don-aot - please can you check the Sector / Subsector fields, they are not behaving correctly, they look correct at the API level so think it is in the page / viewmodel and JS

@AndreGAot  FIxed the error with dropdown

AndreGAot reviewed (approved):
Ready for Dev
